packages/babel-plugin/src/keyframes/__fixtures__/index.tsx

```diff
@@ -0,0 +1,47 @@
+export const longhandCssPropObjectCallExpression = `
```

Comment:
Do we also need to provide examples for `ClassNames`?

Reply:
Added some tests